features/download.yml.dist

```diff
@@ -54,15 +51,12 @@ compat_features:
   - api.HTMLAreaElement.download
 
   # ⬇️ Same status as overall feature ⬇️
-  # baseline: high
-  # baseline_low_date: 2019-09-19
-  # baseline_high_date: 2022-03-19
+  # baseline: false
   # support:
   #   chrome: "14"
   #   chrome_android: "18"
   #   edge: "18"
   #   firefox: "20"
-  #   firefox_android: "20"
   #   safari: "10.1"
   #   safari_ios: "13"
   - html.elements.a.download
```

Comment:
I don't think this is a good change. I've opened https://github.com/mdn/browser-compat-data/pull/25758 to fix this upstream.